Soccar [UI]/src/nl/socnet/message/handler/SpawnObstacleMessageHandler.java

```diff
@@ -0,0 +1,53 @@
+package nl.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.library.Obstacle;
+import nl.soccar.library.Player;
+import nl.soccar.library.enumeration.ObstacleType;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+import nl.soccar.ui.Main;
+import nl.soccar.ui.fx.controller.GameViewFXMLController;
+import nl.soccar.ui.rmi.ClientController;
+import nl.socnet.message.SpawnObstacleMessage;
+
+/**
+ * @author PTS34A
+ */
+public final class SpawnObstacleMessageHandler extends MessageHandler<SpawnObstacleMessage> {
+
+    @Override
+    protected void handle(Connection connection, SpawnObstacleMessage message) throws Exception {
+        Object controller = Main.getInstance().getController();
+        if (controller instanceof GameViewFXMLController) {
+            Player player = ClientController.getInstance().getCurrentPlayer();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "player" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "player". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1854)